src/test/java/co/rsk/bitcoinj/script/StandardRedeemScriptParserTest.java

```diff
@@ -237,14 +230,6 @@ public void getPubKeys_invalid_redeem_script() {
         parser.getPubKeys();
     }
 
-    @Test
-    public void getM_from_no_multiSig() {
```

Comment:
Maybe this is not the place, but we should have some tests like getMFromHardcodedRedeemScriptParser and so on for other methods.

Basically make sure that for the hardcoded redeem script we are getting the expected values